### PR TITLE
Fix broken links in docs

### PR DESCRIPTION
**Pull request by kimauth**

### CODE REVIEW

File: src/PointEval/PointEvalHandler.jl [hunk 1]
```diff
@@ -314,7 +314,7 @@ Create an iterator over the points in the [`PointEvalHandler`](@ref).
 The elements of the iterator are either a [`PointLocation`](@ref), if the corresponding
 point could be found in the grid, or `nothing`, if the point was not found.
 
-A `PointLocation` can be used to query the cell ID with [`cellid`](@ref), and can be used
+A `PointLocation` can be used to query the cell ID with `cellid`, and can be used
```
Comment (KnutAM) [inline src/PointEval/PointEvalHandler.jl:317]:
```suggestion
A `PointLocation` can be used to query the cell ID with the `cellid` function, and can be used
```
Only a suggestion to make it clearer until the docs for `cellid` are available.

**Review by KnutAM — APPROVED** [review]
Nice!
(Would be nice to publish the docstring for AffineConstraint, but that is a separate issue)

File: src/PointEval/PointEvalHandler.jl (hunk 1 above)
Comment (kimauth) [reply src/PointEval/PointEvalHandler.jl:317]: Yeah, not sure if `cellid` should be an explicit part of the API with docs. Perhaps if it gets a different name.

**Comment by kimauth** [comment]
I didn't add the doctoring for `AffineConstraint` to the docs because I wasn't sure if it is meant to be internal.

**Pull request merged**